packages/cli/src/templates/project/package-json.ts

```diff
@@ -37,7 +39,7 @@ export const template = `{
   "scripts": {
     "lint:check": "eslint --ext '.js,.ts' **/*.ts",
     "lint:fix": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
-    "compile": "npx tsc -b tsconfig.json",
+    "compile": "npx ttypescript -b tsconfig.json",
```

Comment:
`[rant]`

Whyyyyyy 😭😭😭😭 Would've been better to use `ts-patch`...

Anyways, go on... 🥲

`[/rant]`

Reply:
I just extended form the existing branch, we'll have plenty of time to change it in the future 🙌🏻